src/quemb/kbe/pfrag.py

```diff
@@ -174,8 +186,9 @@ def sd(
         else:
             raise ValueError(f"Imaginary density in Full SD {max_val}")
 
-        Sites = [i + (nlo * 0) for i in self.AO_per_frag]
-        if not frag_type == "autogen":
+        Sites = [i + (nlo * 0) for i in self.AO_in_frag]
+        # TODO: Discuss with Minsik
```

Comment:
the following sort might be relevant for #144 @mscho527

Reply:
It was already fixed 
https://github.com/troyvvgroup/quemb/pull/144/files#diff-059f900a6633877c395553bbd7e267247aef938a4d8f0b50f4494690a85178aeL172
:-D